packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/ground_overlay.dart

```diff
@@ -244,6 +247,9 @@ class GroundOverlay implements MapsObject<GroundOverlay> {
   /// Callbacks to receive tap events for ground overlay placed on this map.
   final VoidCallback? onTap;
 
+  /// Map zoomlevel used when setting ground overlay with position. iOS only.
```

Comment:
not sure can we say iOS only here as this is plafrom interface.
it might be (technically not possible) that this can be supported by another implementation.
I added the table that has zoomLevel feature compability listed to the app facing api.